msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Together Mode in Teams
+description: Work with Together Mode 
+ms.topic: conceptual
+---
+
+# Together Mode in Teams
+
+Teams Together Mode provides a simple, innovative, and user-friendly meeting environment. It digitally combines participants into a single virtual space instead of being shown separately in boxes. It creates your virtual image and places the image in a virtual environment where you can easily interact with your peers. It makes meetings more engaging in the auditorium view.
+
+> [!VIDEO https://www.youtube-nocookie.com/embed/MGsNmYKgeTA]
+
+You can create scene only apps or extend your current app to have scenes for Together Mode. A scene is a virtual view of the auditorium or meeting venue where the attendees are placed and includes the seats for attendees.
+
+>[!NOTE]
+> Scene only apps are recommended as the acquisition experience for such apps is more seamless.
```

Comment:
Add this statement: 

A scene only app is still an app in Microsoft Teams. The app package creation step is abstracted out since the scene design studio handles the app package creation in the background.

Reply:
Added the statement as a note as the statement is more of additional information.